packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -57,6 +62,27 @@ export class ImportWallet extends React.Component<Props, State> {
   state = {
     backupPhrase: '',
   }
+  didFocusSubscription?: NavigationEventSubscription
+
+  componentDidMount() {
+    this.didFocusSubscription = this.props.navigation.addListener('didFocus', () =>
+      this.checkCleanBackupPhrase()
+    )
+  }
+
+  componentWillUnmount() {
+    if (this.didFocusSubscription) {
+      this.didFocusSubscription.remove()
+    }
+  }
+
+  checkCleanBackupPhrase() {
+    if (this.props.navigation.getParam('clean')) {
+      this.setState({
+        backupPhrase: '',
+      })
+    }
```

Comment:
Do we need it to `setParam({clean: false})` once we've cleared it to avoid unnecessarily cleaning after that step? And possibly frustrating the user if they switch to another app to copy/paste the phrase.

Reply:
Done! thanks :)